charts/dragonfly/values.yaml

```diff
@@ -1218,6 +1218,21 @@ client:
               # location: docker.io
             # - prefix: ghcr.io
               # location: ghcr.io
+        # podman:
+          # # -- configPath is the path of cri-o registries's configuration file.
```

Comment:
cri-o? typo.

Reply:
podman uses `cri-o` and the config is cri-o's config.